src/lib/pipeline.ml

```diff
@@ -140,7 +154,8 @@ module Construct = struct
 
   let hisat fastq = Hisat fastq
 
-  let stringtie bam = Stringtie bam
+  let stringtie ?(use_reference_gtf =`If_available) bam =
```

Comment:
What's the alternative to `If_available` called? 

Also, what do you think of just having a boolean flag for `reference_guided_assembly` and if that's true then a GTF is required.